wheelbase: 0.257 #[m]
width: 0.2
length: 0.5

# box constraints
# velocity
v_max: 0.8 # max long vel [m/s]
v_min: 0 #0.05 # min long vel [m/s]

# steering
delta_min: -0.35  # minimum steering angle [rad]
delta_max: 0.35  # maximum steering angle [rad]

# linear acceleration
a_min: -3.5
a_max: 3.5

# lateral acceleration
alat_max: 5.0

# track parameters
track_file: 'outerloop_center_smooth.csv'
track_width_R: 0.3
track_width_L: 0.3
track_offset: 0.15

# cost
w_vel: 4
w_contour: 70
w_theta: 2
w_accel: 0.15
w_delta: 0.15

# parameters for barrier functions
q1_v: 2
q2_v: 3
q1_road: 1
q2_road: 5
q1_lat: 0
q2_lat: 3
q1_obs: 3
q2_obs: 8.5

# ilqr parameters
T: 1
N: 6  # the number of planning steps
max_itr: 50
# replan_dt: 0.1

